### PR TITLE
Topic on The Passage (book series by Justin Cronin)

### PR DESCRIPTION
**Pull request by Abc2557**
I hope I did this right-
I added a topic I wrote on the book series The Passage by Justin Cronin. If I made any mistakes or you have any suggestions, please tell me!

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -14195,3 +14195,48 @@ label monika_translating_poetry:
...
+
+label monika_thepassage:
+    m 5rub "Have you heard of The Passage?"
+    m 5hub "It's a wonderful book series by Justin Cronin."
+    m 3hksdlb "Well, actually, I haven't finished it yet."
+    m 1sub "But so far, I love it!"
+    m 4eua "It's about a dystopian world filled with people infected by a strange virus which makes them act almost like vampires."
+    m 6wud "But the thing is, they were infected on purpose by scientists."
+    m 1rksdla "It's a bit... hard to explain."
+    m 3hub "Though, it's very interesting. I reccomend you read it!"
```
Comment (Iniquitatis) [inline Monika After Story/game/script-topics.rpy:14219]: "recommend"

Comment (Abc2557) [reply Monika After Story/game/script-topics.rpy]: Done!

**Comment by Abc2557** [comment]
I expect this to take me a while to work on. I feel like I might be holding back the 0.11.3 update (because this PR has the milestone), so I wanted to mention that you can remove that 0.11.3 milestone if you want. I haven’t really been working on it much lately, so... sorry to make you wait... It’s been a long time since I read The Passage, and it’s not one of those books that you can just re-read in a day to get your knowledge back; it’s fairly long. 
I hope to finish rewriting this topic soon, but writer’s block and lack of knowledge may get the best of me, so, if I end up discarding the topic altogether, you shouldn’t be too surprised.

**Comment by ThePotatoGuy** [comment]
understood, marking milestone as soon

**Comment by Abc2557** [comment]
I'm highly considering just closing and scrapping this altogether. I don't plan on working on it anytime soon. It's just a simple topic on a book not a lot of people have heard of. If anyone wants to make a topic on The Passage themself, be my guest. I just don't have nearly enough knowledge on the book, nor do I have the motivation to write a topic on it.

**Pull request closed without merging**